index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html ng-app="ToDoList">
+  <head>
+    <meta charset="utf-8">
+    <title>ToDo list</title>
+    <link href="bower_components/bootstrap/dist/css/bootstrap.css" rel="stylesheet">
+    <script src="bower_components/jquery/dist/jquery.js"></script>
+    <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/angular-resource/angular-resource.js"></script>
+    <script src="js/app.js"></script>
+    <script src="js/toDoListController.js"></script>
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+  </head>
+
+  <body ng-controller="ToDoListController as listCtrl">
+
+    <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
```

Comment:
I'm not sure why you need `role="navigation"` when the element, nav, already communicates that it's for navigation. Is it something to do with bootstrap?